ru-RU/template/isolate-style.md

```diff
@@ -1,3 +1,171 @@
 # Изоляция стилей
 
-TODO
+Изоляция стилей решает проблему конфликта CSS-классов в компонентах.
+Рассмотрим данную проблему на примере двух компонентов, в разметке которых используются CSS-классы с одинаковыми именами, но с разными стилями для каждого из них.
+
+Первый компонент:
+
+`component1/template.tmpl`:
+```html
+<b:style src="./style.css"/>
+
+<div class="some-class">Компонент 1</div>
+```
+
+`component1/style.css`:
+```css
+.some-class {
+  color: red;
+}
+```
+Второй компонент:
+
+`component2/template.tmpl`:
+```html
+<b:style src="./style.css"/>
+
+<div class="some-class">Компонент 1</div>
```

Comment:
Долбаный копипаст....
1 июля 2016 г. 7:50 PM пользователь "Ivan Strelkov" <
notifications@github.com> написал:

> In ru-RU/template/isolate-style.md
> <https://github.com/basisjs/articles/pull/34#discussion_r69327218>:
>
> > +<div class="some-class">Компонент 1</div>
> > +```
> > +
> > +`component1/style.css`:
> > +```css
> > +.some-class {
> > +  color: red;
> > +}
> > +```
> > +Второй компонент:
> > +
> > +`component2/template.tmpl`:
> > +```html
> > +<b:style src="./style.css"/>
> > +
> > +<div class="some-class">Компонент 1</div>
>
> Видимо, лучше всё же здесь по аналогии написать Компонент 2 :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/basisjs/articles/pull/34/files/1aa38bfd977083e8997a2ac03108c097856fb326#r69327218>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGWKdVEme5ugGeWJ3-lWy2mncdzgdMXdks5qRUVFgaJpZM4JDTgH>
> .
>